### PR TITLE
Ofmcc 676 - Supplementary Allowances - initial setup

### PR DESCRIPTION
**Pull request by vietle-cgi**
- Add Funding Agreement request to backend
- Add Supplementary Allowances to frontend

### CODE REVIEW

File: frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue [hunk 1]
```diff
@@ -1,3 +1,210 @@
...
+      <div class="min-height-screen my-4">
+        <v-row v-if="!$route.params.applicationGuid" no-gutters class="my-8">
+          <v-col cols="12" md="5" xl="3" class="pr-4">
+            <AppLabel>To start your application, select a facility:</AppLabel>
+            <div>
+              <v-icon class="mr-1">mdi-information-slab-circle-outline</v-icon>
+              <span>If your facility is not listed, contact your Account Manager.</span>
+            </div>
+          </v-col>
+          <v-col cols="12" md="7" xl="9">
```
Comment (weskubo-cgi) [inline frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue:17]: Can we limit the width of this to something reasonable?

File: frontend/src/views/applications/ApplicationsHistoryView.vue [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <template>
   <v-container fluid>
+    <AppButton id="supp-allowances-button" size="large" width="250px" :to="{ name: 'supp-allowances' }" class="mb-12">Add SUP Application</AppButton>
```
Comment (weskubo-cgi) [inline frontend/src/views/applications/ApplicationsHistoryView.vue:3]: I know there are pending changes to this layout but can we update the name to match the mockup in the meantime?

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/applications/ApplicationsHistoryView.vue (hunk 2 above)
Comment (vietle-cgi) [reply frontend/src/views/applications/ApplicationsHistoryView.vue:3]: I think I follow the same name in this mockup: https://www.figma.com/file/PixIpiMff7mhG6D5SqMu3q/OFM-(New)?type=design&node-id=208-80&mode=design? maybe I look at the wrong one?

Comment (weskubo-cgi) [reply frontend/src/views/applications/ApplicationsHistoryView.vue:3]: Sorry about that. I thought I saw something else in the demo.

File: frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue (hunk 1 above)
Comment (vietle-cgi) [reply frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue]: it is updated.

File: frontend/src/App.vue [hunk 3]
```diff
@@ -10,7 +10,7 @@
       <TheEnvBar />
       <TheModalIdle v-if="isAuthenticated" class="align-start px-8 mb-0" />
       <TheHeroImage v-if="$route.meta.showHeroImage" />
-      <TheFacilityHeader v-if="isActingProvider && !isApplicationPage" :showFacility="$route.meta.showFacility" />
+      <TheFacilityHeader v-if="isActingProvider && !$route.meta.hideFacilityHeader" :showFacility="$route.meta.showFacility" />
```
[the remark this answers is not in the document]
Comment (vietle-cgi) [reply frontend/src/App.vue:13]: sounds good. thank you!

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**